cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,70 @@
+Cypress.on("uncaught:exception", (err, runnable) => {
+  return false;
+});
+
+describe("test calculator", () => {
+  const handleClickDigit = (num) => {
+    cy.get(".digit").contains(num.toString()).click();
+  };
+
+  const handleClickOperator = (operator) => {
+    cy.get(".operation").contains(operator).click();
+  };
+
+  before(() => {
+    cy.visit("http://127.0.0.1:5500/");
+  });
```

Comment:
그리고 다른 테스트가 통과하지 못하는 이유는,
첫번째 테스트가 통과된 다음에
UI가 초기화되지 않기 때문입니다 ㅎㅎ


```suggestion
  beforeEach(() => {
    cy.visit("./index.html");
  });
```
그래서 이렇게 `beforeEach`로 변경하면 된답니다!
before는 전체 테스트가 실행되기 전에 한 번 실행하는 함수고
beforeEach는 각각의 테스트가 실행되기 전에 실행해주는 함수입니다.